applications/RomApplication/tests/NonlinearTestFiles/MainKratosROM.py

```diff
@@ -0,0 +1,187 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.ConvectionDiffusionApplication
+import KratosMultiphysics.RomApplication as romapp
+
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.apply_thermal_face_process import ApplyThermalFaceProcess
+from convection_diffusion_analysis import ConvectionDiffusionAnalysis
```

Comment:
this will stop working in october, please always use the full name (i.e. include the app, same as what you do two lines above)